docs/config_reference.rst

```diff
@@ -1923,6 +1923,15 @@ A *processor info object* in ReFrame's configuration is used to hold information
 
    .. versionadded:: 4.6
 
+.. attribute:: systems.partitions.processor.platform
+
+   :required: No
+   :default: ``None``
+
+   The OS platform for this processor (e.g., ``x86_64``, ``arm64`` etc.)
```

Comment:
Not sure this is really OS related. The Python docs just call this the ["machine type."](https://docs.python.org/3/library/platform.html#platform.machine)

Reply:
I could update the docs, but do you think we should also change the name of the actual option?

Reply:
> do you think we should also change the name of the actual option?

Now that you mention it, what do you think about `instruction_set`?

I know I'm nit picking here... Because there's a `platform.platform()` and a `platform.machine()`, I don't like the idea of crossing them. Maybe the name `machine` is better? It would at least map "machine" to `platform.machine()`.

Reply:
I would prefer to go with something that is used more generally for this string. For example, Docker calls this "platform", Linux's `uname` calls it "machine" but also in the docs (for MacOS) describes as "hardware platform". That's why I have used the term "platform" in first place, but I'm open also for "machine" or "machine_type".

Reply:
OK good points 😄 , I'm good with "platform", but would still recommend dropping OS from the docs.